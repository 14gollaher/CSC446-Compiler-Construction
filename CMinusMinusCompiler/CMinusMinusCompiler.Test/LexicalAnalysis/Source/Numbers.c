123
123.45
9 8 7.6
