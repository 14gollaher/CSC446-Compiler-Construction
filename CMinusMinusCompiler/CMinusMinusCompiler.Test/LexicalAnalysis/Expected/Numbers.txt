Lexeme                              Token                Attributes     
-------------------------------------------------------------------
Wheee                               IdentifierToken                    
Hello                               IdentifierToken                     
wORlD                               IdentifierToken                     
abcdefghijklmnopqrstuvwxyz_         IdentifierToken                     
abcdefghijklmnopqrstuvwxyz123       IdentifierToken      abcdefghijklmnopqrstuvwxyz1
abcdefghijklmnopqrstuvwxyz123       IdentifierToken      abcdefghijklmnopqrstuvwxyz1
if                                  IfToken                             
ELse                                IdentifierToken                     
hello                               IdentifierToken                     
while                               WhileToken                          
\n                                  EndOfFileToken                      
